### PR TITLE
Math Magicians : Components

### PR DESCRIPTION
**Pull request by sja-thedude**
- Created a directory called components
- Created a new Calculator.js file in components
- Imported using App.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
+/* eslint-disable react/prefer-stateless-function */
```
Comment (rloterh) [inline src/App.js]: - [x] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is quality and clean.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Syeda, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Stateful component successfully created
- UI matches design guideline
- App is running correctly
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (sja-thedude) [reply src/App.js]: @rloterh Sir, I have enabled and fixed the errors.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @sja-thedude, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**